packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -102,49 +94,80 @@ export const AddToCartModal = () => {
                             flex="1"
                             paddingRight={{lg: '4', xl: '8'}}
                             paddingY={{base: '4', lg: '0'}}
+                            maxHeight="280px"
+                            overflowY="scroll"
```

Comment:
Still need to work on the styling work here. At the moment I simply added a max height and scrolling to where we normally had the product item component, which now has the possibility for multiple products.

Reply:
Nate, Phil, and I had a chat about the height of the modal when it is a product set here, we agreed that we don't need to change anything here. The current implementation is that the height of the modal is spilled over the height of the screen most of the time, and users just scroll the outside page down to see the modal. We can just keep it that way since the detailed styling will depend on customer need.

Reply:
Thanks for pointing that out. It's been removed.